### PR TITLE
Deploy to multiple Ethereum networks

### LINKED ISSUE #64: Prepare the subgraph to be deployed to multiple Ethereum networks
Reference: https://thegraph.com/docs/en/hosted-service/deploy-subgraph-hosted/#deploying-the-subgraph-to-multiple-ethereum-networks

### PR DESCRIPTION
**Pull request by llunaCreixent**
- [x] https://github.com/CirclesUBI/circles-subgraph/issues/64
- [x] Update package-lock.json

Closes https://github.com/CirclesUBI/circles-subgraph/issues/64

### CODE REVIEW

File: .env-poa-sokol.example [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+ACCESS_TOKEN=<YOUR TOKEN>
+IS_DEBUG=false
+
+# This is from when we deployed the Hub contract on poa-sokol
+START_BLOCK=0
+
+# Manifest variables
+HUB_ADDRESS=0x5D0dDd1d35D3FA9950Defd99F92053b3B9E236D0
+PROXY_FACTORY_ADDRESS=0x0F9f9C38E53674EF7a29A1DE38EAbF34e1DCB81C
```
Comment (JacqueGM) [inline .env-poa-sokol.example]: EOF

File: .env-poa-sokol.example [hunk 2]
```diff
@@ -0,0 +1,15 @@
+SUBGRAPH_NAME=circlesubi/circlesubi-staging
+SUBGRAPH_NETWORK=poa-sokol
+
+GRAPH_ADMIN_NODE_ENDPOINT=https://api.thegraph.com/deploy/
```
Comment (JacqueGM) [inline .env-poa-sokol.example:4]: no seria sin deploy  o por que esta el deploy ahi?

File: .env-xdai.example [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+ACCESS_TOKEN=<YOUR TOKEN>
+IS_DEBUG=false
+
+# This is from when we deployed the Hub contract on xDai
+START_BLOCK=12529458
+
+# Manifest variables
+HUB_ADDRESS=0x29b9a7fBb8995b2423a71cC17cf9810798F6C543
+PROXY_FACTORY_ADDRESS=0x8b4404DE0CaECE4b966a9959f134f0eFDa636156
```
Comment (JacqueGM) [inline .env-xdai.example]: EOF

File: .env-poa-sokol.example (hunk 2 above)
Comment (llunaCreixent) [reply .env-poa-sokol.example:4]: This is the endpoint we've been using. If it doesn't work we'll update it.

Comment (llunaCreixent) [reply .env-poa-sokol.example:4]: This endpoint will be compatible with the self-hosted graph-node too.

Comment (JacqueGM) [reply .env-poa-sokol.example:4]: but I have seen this one in various  config files
<graph:
  # Graph node URL
  endpoint: https://api.thegraph.com without the `deploy`

Comment (llunaCreixent) [reply .env-poa-sokol.example:4]: Can you link me to the files?

Comment (JacqueGM) [reply .env-poa-sokol.example:4]: https://github.com/CirclesUBI/circles-iac/blob/main/helm/circles-infra-suite/values.yaml#L61

Comment (llunaCreixent) [reply .env-poa-sokol.example:4]: Note that here we need the admin api, not the public query api

**Review by JacqueGM — APPROVED** [review]

**Pull request merged**